src/component/calc.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calc extends React.Component {
+  render() {
```

Comment:
- Great job rendering the calculator. 💪🏽 
- I'd like to kindly suggest that we follow the project requirements by using a `Functional` component instead of a `Class` component.

<img width="523" alt="Screenshot 2023-03-22 at 14 48 47" src="https://user-images.githubusercontent.com/99246359/227020649-6545767a-d20b-4969-aab1-152f8fd1da61.png">

- Also, I'd like to kindly suggest that we remove all commented lines of code to make the code base much more readable.